### ISSUE TITLE
Problem with command initializing

### ISSUE THREAD

**Issue by a6avind** [LuD1161/upi-recon-cli#1]
Hi Team,
I was trying to play around with the tool and came across an error while initializing the cli.

Am i doing something wrong here?

![image](https://user-images.githubusercontent.com/11615112/168475648-e3e9f2fa-2c2d-41ae-bb93-e0eaf8afbd98.png)

Do let me know, if you require any other details to recreate the problem.

**Comment by LuD1161** [comment]
Hey @a6avind  👋 
Thanks for trying out the tool.
Check the installation guide [here](https://github.com/LuD1161/upi-recon-cli#installation)
> Rename the config.yaml.sample file to config.yaml and add the razorpay live api key into it. Please [refer to the documentation](https://razorpay.com/docs/payments/dashboard/settings/api-keys/) provided by Razorpay in order to generate valid live API credentials.
Note: Razorpay test credentials will not work.

You need to add the razorpay live credentials into `config.yaml` file in the same directory.

**Comment by a6avind** [comment]
Hey @LuD1161,
Got confused with the errors in the initialization. Is it supposed to happen without the keys?
Will check the documentation.

Cheers.

**Comment by LuD1161** [comment]
Yup. If you don't have keys, that's when it's supposed to happen. Will put in a proper error message for the same 🙌 

Thanks for pointing this out 😄

**Comment by LuD1161** [comment]
Hey @a6avind 
Fixed the issue. Can you confirm 🤔 ?

If it still exists, feel free to reopen this one 😄 

Actually merging the PR with referencing this issue, automatically closed this one.

**Comment by a6avind** [comment]
Hey @LuD1161 
It looks like the issue has been resolved.

**Comment by LuD1161** [comment]
Thanks 😄 🙌

**Comment by a6avind** [comment]
One more suggestion, before starting to checks for VPAs, we can look whether the API key is valid or not.

Cheers.

**Comment by LuD1161** [comment]
Hey @a6avind 👋 
Can you raise a new issue for the same will help in tracking 🙌 😄

**Comment by a6avind** [comment]
okay
